src/app/api/webhooks/tips/sanity/route.ts

```diff
@@ -38,6 +40,14 @@ export async function POST(req: NextRequest) {
       const {_id, title, slug, instructorId} = sanityRequestBody.data
       console.info('processing Sanity webhook: Lesson created', _id)
 
+      await inngest.send({
+        name: SEND_SLACK_MESSAGE_EVENT,
+        data: {
+          instructorId,
+          message: `_Created egghead tip_: *<https://egghead.io/tips/${slug.current}|${title}>*\n\n`,
```

Comment:
passing in message here so we can re-use this event for other instances (like egghead-feedback)